.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Run-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Tests
+        run: bash tests/run_integration_tests.sh
+  openshift-yaml-validate-pgbouncer:
```

Comment:
the steps are repeated you can use a matrix startergy here

Reply:
*Strategy

Reply:
Thank you @sawood14012. will make the required changes